### DISCUSSION TITLE
Face used for fringe

### DISCUSSION THREAD

**Discussion by butter-radish** [minad/vertico#468] [Q&A]
I was trying to theme Vertico with Solaire mode. Unfortunately, I’m getting into some trouble figuring out the face that Vertico uses for the fringes with continuation lines; I’m using Olivetti so my fringes are much larger.

![Screenshot_2024-04-13_21-21-55](https://github.com/minad/vertico/assets/92060841/ab7df4c4-25e4-490b-a6a6-ccffa70dbed5)

The face in question is the pale yellow streaks used for the fringes where entries were truncated by Vertico. As you can see, only continuation lines have the yellow face; the blue is what I want it to look like, but I need to figure out what face that is for that. It’s not the ‘fringe’ face like the non-continuation lines.

**Comment by minad** [comment]
I am afraid you won't get an answer here. Maybe ask the author of Solaire mode or the author of your theme?

**Reply by butter-radish** [reply]
Sorry for not being clear in the first post: I wrote the theme, and Solaire mode needs to know the face being used (by Vertico) to switch it over the way I want. I wasn’t sure who else would know the face that Vertico is using in this instance. ☹️

I know that it’s neither ‘fringe’ nor ‘olivetti-fringe’, because those are already correctly being switched over by Solaire (that’s the blue margins on the non-continuation lines).

I’m also using Marginalia; could this be a Marginalia face? The names of none of those had “fringe” or “margin” in them.

**Reply by minad** [reply]
Tbh I don't know which face it is.

**Reply by butter-radish** [reply]
Aw. But that’s good, it means this is probably something standard? I’ll try asking Solaire’s dev if they have any idea what faces could be used for fringes in minibuffers. Thenks!
